pkg/kv/key.go

```diff
@@ -404,6 +406,9 @@ type strHandleVal struct {
 	val interface{}
 }
 
+const sizeofHandleMap = int64(unsafe.Sizeof(*(*HandleMap)(nil)))
```

Comment:
We need to track the memory usage of the underlying structure of golang map

refer: https://github.com/pingcap/tidb/blob/master/pkg/kv/key.go#L527